piqueserver/config/config.toml

```diff
@@ -168,6 +168,12 @@ time_announcements = [ 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 30, 60, 120, 180, 240, 300
 # number of intel captures to win the game (default: 10)
 cap_limit = 10
 
+# Enable command antispam for preventing users to abuse from flooding commands
+enable_command_antispam = true
```

Comment:
It may be out-of-scope for this PR, though having a common configuration section for antispam features might make the config more consistent.

Reply:
I believe they're also called 'ratelimits' internally - so we may also want to rename this to 'enable command rate-limiter' or something along those lines. If we do that then also make sure to rename the other settings.